docs/api/migration-guides/qiskit-1.0.mdx

```diff
@@ -0,0 +1,594 @@
+
+  This is not the same object that you `pip install` (which is a *distribution*), but commonly the thing you `pip install` and the thing you `import` will have the same name.
+
+* *submodule*, *subpackage*: These are somewhat imprecise terms, but are commonly used.
+  The *sub* part just means "contained inside a package".
+  A *submodule* is just a module and a *subpackage* is just a package, but they are part of a larger *package*.
+
+* *namespace package*: A *package* that can have more submodules or subpackages installed into it by other *distributions*.
+  Critically, no one distribution contributing to a namespace package necessarily owns all the installed files, so it can be tricky to completely uninstall
+  or upgrade one.
+
+* *distribution*: The compressed Python files, data files, and metadata that are downloaded when you run `pip install something`.
+  Often, a *distribution* contains exactly one *package* and the metadata about how to install it (its requirements, and so on), but this is not a necessity.
+  A *distribution* can contain zero or more *modules* or *packages*.
+
+  If you are familiar with "package managers" outside the context of Python, such as Debian/Ubuntu's `apt` or macOS's Homebrew, then what they call a "package", Python instead calls a *distribution*, and there is no exact match for what Python calls a *package*.
+  Python is the unusual one in this regard.
+
+  Most sources talking about Python packaging will use *package* to mean both *distribution* and *package*, and you have to understand from context what is meant.
+  In general, if you `import` it, they meant "package" and if you `pip install` it, they meant "distribution".
+
+* *search path*: When trying to `import something`, Python will search a pre-defined list of places for a *module* or *package* called `something`.
+  The list of places is the *search path*.
+  You can see and modify the search path in `sys.path`.
+
+* *requirement*: A *distribution* contains information on other distributions it depends on when installed.
+  Any other distribution that is necessary is a *requirement*, and the package manager (usually `pip` or `conda`) should ensure that all requirements are installed with compatible versions.
+
+Python is highly dynamic, and it's actually possible for a *module* or *package* not to correspond to files on disk at all, or to be compiled extensions, or many other things.
+The *search path* is not only a search over directories—that is only one component—but files on disk are all that matters here.
+These complications aren't necessary to understand the problems of this section, so you can use the model described above.
+</Admonition>
+
+Historically, Qiskit comprised of many Python distributions: `qiskit-terra`, the compiler core; `qiskit-aer`, the high-performance simulator; the original IBM Quantum provider; and several now-obsolete packages providing particular exploratory algorithmic or error-mitigation features.
```

Comment:
ignis was actually more about running characterization experiments (like qiskit-experiments now) rather than error mitigation.

Reply:
Thanks - the only bits I ever interacted with were the error-mitigation bits.  Changed in 75f6e3a.